packages/cli/src/controllers/oauth/oAuth2Credential.controller.ts

```diff
@@ -87,6 +81,7 @@ export class OAuth2CredentialController extends AbstractOAuthController {
 	/** Verify and store app code. Generate access tokens and store for respective credential */
 	@Get('/callback', { usesTemplates: true })
 	async handleCallback(req: OAuthRequest.OAuth2Credential.Callback, res: Response) {
+		const userId = req.user?.id;
 		try {
 			// realmId it's currently just use for the quickbook OAuth2 flow
```

Comment:
Unrelated, but we're not getting the realmId from the query. Is this comment still accurate?

Reply:
looks like this comment has been moved from file to file since [it was added 3 years ago](https://github.com/n8n-io/n8n/pull/1365/files#diff-e1c41b1f8cc627242377b30fbee7af28472c81abe5f4f41da1c513f264f128e2R1315).
The comment does not make sense in that PR either.
I'll delete it.